File: 1905.py
import sys
sys.setrecursionlimit(100000)

n = 1
total = 0


def test(num):
    global n
    global total
    total += n
    n += 1
    if n <= num:
        test(num)

    return total


num = int(input())

print(test(num))

File: 1905.cpp
#include <iostream>
using namespace std;

int n = 1;
int total = 0;

int test(int num)
{
    total += n;
    n++;
    if (n <= num)
    {
        test(num);
    }
    return total;
}

int main()
{
    int num;
    cin >> num;

    cout << test(num);

    return 0;
}